ui/src/app/modules/projects/modules/integrations/components/integration/integration.component.html

```diff
@@ -1,7 +1,23 @@
 <div class="integration-screen">
-
-    <div class="integration-tab">
-        <h1>Integrations</h1>
+    <div class="steps">
+        <h1><strong> Step 1 :</strong> Create a <strong> Project Logs </strong> project</h1>
+        <mat-card>
+            <p>
+                Before you can add <strong> Project Logs </strong> to your app, you need to create a new <strong> Project Logs </strong> project to connect to your app. Please in the left panel click on New Project.
```

Comment:
improve text like this

- On the left menu you will see a link to create your projects